test/sys/test_aio.rs

```diff
@@ -26,6 +26,7 @@ fn poll_aio(mut aiocb: &mut AioCb) -> Result<()> {
 // bindings.  So it's sufficient to check that AioCb.cancel returned any
 // AioCancelStat value.
 #[test]
+#[cfg_attr(all(target_env = "musl", target_arch = "x86_64"), ignore)]
```

Comment:
But I've filed an issue (japaric/cross#107) in order for this to be made more clear.